src/Resources/config/config.xml

```diff
@@ -141,4 +141,15 @@
             <label>Order Shipping State</label>
         </component>
     </card>
+    <card>
+        <title>Payment Flow</title>
+        <input-field type="bool">
```

Comment:
Will this be On by default? Would it be enabled for merchants upgrading?

Reply:
No they would have to select it. Default is the regular async flow